os/src-jvm/package.scala

```diff
@@ -38,7 +39,9 @@ package object os {
   /**
    * The current working directory for this process.
    */
-  val pwd: Path = os.Path(java.nio.file.Paths.get(".").toAbsolutePath)
+  def pwd: Path = pwd0.value
+  val pwd0: DynamicVariable[Path] =
```

Comment:
Can we find a better name? `pwdContext` or `pwdDynamic`?

Would be nice, if reading code gives a clue what's happening.

Example:

```scala
val y = os.pwd0.withValue(os.pwd / "hello") {
  os.pwd
}
```

vs.

```scala
val y = os.pwdContext.withValue(os.pwd / "hello") {
  os.pwd
}
```

Reply:
I went with `dynamicPwd`